package-dev/Runtime/SentryInitialization.cs

```diff
@@ -37,17 +38,30 @@ public static void Init()
             {
                 var sentryUnityInfo = new SentryUnityInfo();
 
+                Exception nativeInitException = null;
```

Comment:
I wonder why we didn't have anything like that, as is, if any of the `SentryNative*.Configure()` throws for whatever reason, that exception is not captured, **nor are any others because `SentryUnity.Init()` is never called**
https://sentry.io/organizations/sentry-sdks/issues/3259732515/events/5f72ce01d1074bfb97c59c6fa0a1b7df/?project=5439417

Reply:
I've improved the exception text here: https://sentry.io/organizations/sentry-sdks/issues/3259732515/?project=5439417&query=is%3Aunresolved